languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/server/ServerRenderer.kt

```diff
@@ -116,8 +117,9 @@ class ServerRenderer @Inject constructor(
                         |            ${if(it.hasQueryParams()) "queryParams: request.query as any," else ""}
                         |            <${if(it.hasBody()) "body: request.payload as any" else ""}>
                         |          });
+                        |          ${it.removeUnknownProperties()}
```

Comment:
Here we render the code to strip unknown properties. It will store the result in the new variable `body`.